CMakeLists.txt

```diff
@@ -331,7 +334,10 @@ install(DIRECTORY
   USE_SOURCE_PERMISSIONS
 )
 
+install(DIRECTORY include/ DESTINATION include)
+
 ament_export_include_directories(include)
 ament_export_libraries(${library_name})
```

Comment:
Why not add to export libraries? It should already be being done from this line

Reply:
I would probably reject this PR @ayrton04 specific user modifications for non-standard applications is exactly the reason to fork and I'm not sure introducing this has any benefit to us as maintainers / users moving forward

Reply:
So I agree with that sentiment for the extra `export` and enabling PIC, but it seems like we install the `include` directory in ROS 1:

https://github.com/cra-ros-pkg/robot_localization/blob/87b8d6bf6c94933da9f2f830460fa1c38c651db5/CMakeLists.txt#L208-L212

Thoughts?

Reply:
The installing headers is fine, its the exports

Reply:
@firemark I am inclined to agree with @SteveMacenski on this. We can install the header files, but I'd prefer not to have the export. Not sure how I feel about enabling PIC.

Reply:
Any update here? If we can remove the export and PIC for now, I'd appreciate it.

Reply:
I'm sorry for a late response - I don't have time to finish the PR. I leave only line with installing includes.